src/pipelines.py

```diff
@@ -1,13 +1,14 @@
 from functools import partial
 
-from loaders import MetadataImageSegmentationLoader, MetadataImageSegmentationMultitaskLoader, \
+from .steppy.base import Step, Dummy
+from .steppy.preprocessing.misc import XYSplit, ImageReader
+
+from .loaders import MetadataImageSegmentationLoader, MetadataImageSegmentationMultitaskLoader, \
     ImageSegmentationMultitaskLoader, ImageSegmentationLoader
-from models import PyTorchUNet, PyTorchUNetMultitask
-from postprocessing import Resizer, Thresholder, NucleiLabeler, Dropper, \
+from .models import PyTorchUNet, PyTorchUNetMultitask
+from .postprocessing import Resizer, Thresholder, NucleiLabeler, Dropper, \
```

Comment:
@taraspiotr I am pretty sure that WateshedContour is the only postprocessing needed (maybe BinaryFillHoles but nothing else)